tests/distro-regression/tests/bundles/focal-ussuri-ovn-22.03.yaml

```diff
@@ -264,6 +264,7 @@ applications:
       openstack-origin: *openstack-origin
       spare-pool-size: 2
       loadbalancer-topology: 'ACTIVE_STANDBY'
+    constraints: mem=4096
```

Comment:
octavia-diskimage-retrofit needs the constraint too

Reply:
I think octavia-diskimage-retrofit is a subordinate to glance-simplestreams-sync.

Reply:
it is a subordinate, although it doesn't seem to have an explicit relation to g-s-s https://charmhub.io/octavia-diskimage-retrofit/integrations - it uses the juju-info relation, so maybe we should put the constraint on g-s-s ?

```
- - glance-simplestreams-sync:juju-info
  - octavia-diskimage-retrofit:juju-info
```

Reply:
So the reason for why I bumped octavia to mem=4096 is because octavia was hitting OOM errors. While it probably wouldn't hurt much to bump gss or octavia-diskimage-retrofit to 4096, it seems unrelated to the octavia OOM issues. I'm probably being thick-headed and missing something here?

Reply:
my idea that this is needed comes from [this change](https://github.com/openstack-charmers/charmed-openstack-tester/pull/128/files#diff-20a4a39bca0397c92e9df2e36eabf5996e38f717e7dca33818fa1ec1de67ece5R299) which adds the constraint to the retrofit charm, wasn't this intended?

Reply:
Ahh... got it. That is a mistake. Let me remove it.